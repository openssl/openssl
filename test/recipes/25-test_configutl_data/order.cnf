# vim:ft=conf
openssl_conf = openssl_init

[def]
# Sections are alphabetically ordered
0.recipient = 0

[abc]
# Order within sections is preserved, even if it isn't sorted
7.recipient = 7
6.recipient = 6
4.recipient = 4
3.recipient = 3
5.recipient = 5
2.recipient = 2
1.recipient = 1
0.recipient = 0

[default]
# The default section is consolidated and always printed first
aaatest = value
