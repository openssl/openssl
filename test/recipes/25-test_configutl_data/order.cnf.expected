openssl_conf = openssl_init
aaatest = value

[abc]
7.recipient = 7
6.recipient = 6
4.recipient = 4
3.recipient = 3
5.recipient = 5
2.recipient = 2
1.recipient = 1
0.recipient = 0

[def]
0.recipient = 0
